pom.xml

```diff
@@ -158,6 +149,7 @@
             <Bundle-SymbolicName>nu.validator.htmlparser</Bundle-SymbolicName>
             <Bundle-Version>${project.version}</Bundle-Version>
             <Bundle-RequiredExecutionEnvironment>J2SE-1.5</Bundle-RequiredExecutionEnvironment>
+            <Automatic-Module-Name>nu.validator.htmlparser</Automatic-Module-Name>
```

Comment:
As explained in https://github.com/validator/htmlparser/issues/17#issuecomment-653795447, I believe this should be reverted.

Reply:
I'm fine with this, no need to revert (see https://github.com/validator/htmlparser/issues/26)